.github/workflows/continous-integration.yml

```diff
@@ -337,7 +337,10 @@ jobs:
         DD_ARGS: --ddtrace --ddtrace-patch-all
       run: |
         make ${{ matrix.test }}
-        poetry run coverage xml -o ${{ github.workspace }}/tests/${{ matrix.test }}-coverage.xml --include="rasa/*"
+        if [[ "${{ matrix.os }}" != "windows-2019" ]]; then
+          mv .coverage ${{ github.workspace }}/${{ matrix.test }}-coverage
+        fi
+      shell: bash
```

Comment:
I think this is worth a comment. For simplicity, wouldn't it be better to use `bash` in all the steps? (maybe that'd require more work though)

Reply:
There are some other places (even before this PR) where we use shell for some steps. 
I can comment that the bash shell is a way to make code run for both Linux and Windows (to avoid our code having to be Windows-Powershell-compatible)

Reply:
ok sounds good 👍🏻